dockerfiles/ubuntu/ubuntu-16.04/ubuntu-16.04-base/Dockerfile

```diff
@@ -17,7 +17,7 @@
 FROM ubuntu:16.04
 
 RUN apt-get update && \
-    apt-get install -y \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
Is this needed due to the tzdata addition? If so, could you add the reason to the commit message?

Reply:
The `tzdata` addition triggered a failure without this but in general this should be there incase other packages acquire interactive questions in the future.

Reply:
Ok, then could put that in the body of the message?

Reply:
Will do.